website/content/commands/operator/raft.mdx

```diff
@@ -104,3 +104,29 @@ Usage: `consul operator raft remove-peer -address="IP:port"`
 - `-id` - ID of the server to remove.
 
 The return code will indicate success or failure.
+
+## transfer-leader
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/operator/raft/transfer-leader](/api-docs/operator/raft#transfer-leader)
+
+This command transfer the Consul leader to another server. If an `id` is provided, raft leadership will be transferred
+to the server with that id.
+
+[`consul members`](/commands/members) command, it is preferable to
+clean up by simply running
+[`consul force-leave`](/commands/force-leave)
+instead of this command.
+
+The table below shows this command's [required ACLs](/api-docs#authentication). Configuration of
+[blocking queries](/api-docs/features/blocking) and [agent caching](/api-docs/features/caching)
+are not supported from commands, but may be from the corresponding HTTP endpoint.
+
+| ACL Required     |
+| ---------------- |
+| `operator:write` |
+
+Usage: `consul operator raft transfer-leader -id="server id"`
+
+- `-id` - ID of the server to transfer leadership to.
+
+The return code will indicate success or failure.
```

Comment:
Do we mention this for other commands? Or is this implied (not stated) because it's true for all commands?

Reply:
I copy pasted this from another command just above